pom.xml

```diff
@@ -38,6 +36,9 @@
                 <plugin>
                     <artifactId>maven-compiler-plugin</artifactId>
                     <version>${maven-compiler-plugin.version}</version>
+                    <configuration>
+                        <release>${release.version}</release>
```

Comment:
The -release argument for the Java compiler is supported in Java 9+, but we still need to use Java 8 here.

Reply:
True, it's bit more complicated according to this post.
https://stackoverflow.com/questions/59049980/maven-compiler-release-as-an-replacement-for-source-and-target

Maybe provide official releases targeted for Java8?